<?xml version="1.0"?>
<launch>
	<!-- Semantic Segmentation node  -->
	<node type="image_bbox_to_3d_static.py" name="imageBBoxTo3D_Static" pkg="image_boundingbox_to_3d" respawn="true" output="screen">
		<param name="topicCamImage" value="/Multisense/left/image_rect_color" />
		<param name="topicCamInfo" value="/Multisense/left/image_rect_color/camera_info" />
		<param name="topicBBoxIn" value="/yolo/BBox/Multisense" />

		<param name="topicBBoxOut" value="/yolo/BBox3d_static/Multisense" />
		<param name="topicVisualizeOut" value="/yolo/BBox3dImage_static/Multisense" />

		<param name="cam_height" value="2.056" /> 
		<param name="cam_tilt" value="0.33233" /> 
		<param name="cam_fov_vertical" value="0.875" /> 
		<param name="cam_fov_horisontal" value="1.442" /> 

		<param name="visualizeBoundingboxes" value="False" /> <!-- Select either False: No visualization topic is published, True: A image for visualizing bounding boxes are published  -->
	</node>
</launch>
